### DISCUSSION TITLE
iCloud and Audiobookshelf

### DISCUSSION THREAD

**Discussion by oboe45** [advplyr/audiobookshelf#2232] [Q&A]
Could someone post a short set of instructions on how to set up audiobokshelf so that it works with required files in iCloud? I have an iCloud subscription with lots of free space.
 My brain cannot figure out to do this without some help!  Probably a no-brainer for most folks, but not for me.

**Comment by st0r4ge** [comment]
Theoretically, it **_could_** be possible to **_somehow_** mount your iCloud cloud folder on a Linux system (e.g., /mnt/cloud/my-icloud) and then have the Docker container point to this location. Although my quick 2-minute Google search did not show up any simple solutions.

There's only an older script, which most likely won't work anymore:
https://github.com/samuelthomas2774/icloud-drive-fs

And a Snap application without further documentation on mounting, etc.:
https://github.com/cross-platform/icloud-for-linux

Then again, what are your storage requirements? Is your plan to have the cloud storage as a backup solution? After all, to run Audiobookshelf, you already need to have some kind of system with storage.

For backup there is some discussion on the rclone github and forums:
https://forum.rclone.org/t/rclone-vs-icloud/18180
https://github.com/rclone/rclone/issues/1778
